pkg/api/handlers/libpod/play.go

```diff
@@ -70,6 +70,16 @@ func PlayKube(w http.ResponseWriter, r *http.Request) {
 		password = authConf.Password
 	}
 
+	logDriver := query.LogDriver
+	config, err := runtime.GetConfig()
+	if err != nil {
+		utils.Error(w, http.StatusBadRequest, err)
```

Comment:
```suggestion
		utils.Error(w, http.StatusInternalServerError, err)
```
As it is unrelated to the request.